data/xp-pen-artist-22r-pro.tablet

```diff
@@ -0,0 +1,27 @@
+# XP-Pen
+# Artist 22R Pro
+
+[Device]
+Name=XP-Pen Artist 22R Pro
+ModelName=
+DeviceMatch=usb|28bd|091b|UGTABLET 21.5 inch PenDisplay Pen;usb|28bd|091b|UGTABLET 21.5 inch PenDisplay Pad
+PairedIDs=
+Class=Bamboo
```

Comment:
```suggestion
Class=Cintiq
```
is probably a better choice here since it's a display tablet. not that it matters much...

Reply:
Changed, but I wonder - what does this control exactly?

Reply:
Ah this is deprecated now :)

Reply:
the only use-case for this is is still `Class=Remote` but see #779. Still better to have it *somewhat* correct.

Come to think of it: we could probably generate this generically -> https://github.com/linuxwacom/libwacom/issues/790